appservice/src/createAppService/AppServicePlanCreateStep.ts

```diff
@@ -21,29 +20,28 @@ export class AppServicePlanCreateStep extends AzureWizardExecuteStep<IAppService
             const createdAppServicePlan: string = localize('CreatedAppServicePlan', 'Successfully created App Service plan "{0}".', newPlanName);
-            await window.withProgress({ location: ProgressLocation.Notification, title: findingAppServicePlan }, async (): Promise<void> => {
-                ext.outputChannel.appendLine(findingAppServicePlan);
-                const client: WebSiteManagementClient = new WebSiteManagementClient(wizardContext.credentials, wizardContext.subscriptionId, wizardContext.environment.resourceManagerEndpointUrl);
-                addExtensionUserAgent(client);
-                const rgName: string = nonNullValueAndProp(wizardContext.resourceGroup, 'name');
-                const existingPlan: AppServicePlan | undefined = <AppServicePlan | undefined>await client.appServicePlans.get(rgName, newPlanName);
-                if (existingPlan) {
+            if (wizardContext.progress) {wizardContext.progress.report({message: findingAppServicePlan}); }
+            ext.outputChannel.appendLine(findingAppServicePlan);
+            const client: WebSiteManagementClient = new WebSiteManagementClient(wizardContext.credentials, wizardContext.subscriptionId, wizardContext.environment.resourceManagerEndpointUrl);
+            addExtensionUserAgent(client);
+            const rgName: string = nonNullValueAndProp(wizardContext.resourceGroup, 'name');
+            const existingPlan: AppServicePlan | undefined = <AppServicePlan | undefined>await client.appServicePlans.get(rgName, newPlanName);
+            if (existingPlan) {
                     wizardContext.plan = existingPlan;
-                    window.showInformationMessage(foundAppServicePlan);
+                    if (wizardContext.progress) {wizardContext.progress.report({message: foundAppServicePlan}); }
                     ext.outputChannel.appendLine(foundAppServicePlan);
                 } else {
+                    if (wizardContext.progress) {wizardContext.progress.report({message: creatingAppServicePlan}); }
                     ext.outputChannel.appendLine(creatingAppServicePlan);
-                    window.showInformationMessage(creatingAppServicePlan);
                     wizardContext.plan = await client.appServicePlans.createOrUpdate(rgName, newPlanName, {
                         kind: getAppServicePlanModelKind(wizardContext.newSiteKind, nonNullProp(wizardContext, 'newSiteOS')),
                         sku: nonNullProp(wizardContext, 'newPlanSku'),
                         location: nonNullValueAndProp(wizardContext.location, 'name'),
                         reserved: wizardContext.newSiteOS === WebsiteOS.linux  // The secret property - must be set to true to make it a Linux plan. Confirmed by the team who owns this API.
                     });
-                    window.showInformationMessage(createdAppServicePlan);
+                    if (wizardContext.progress) {wizardContext.progress.report({message: createdAppServicePlan}); }
```

Comment:
Can you do a gif of this experience rather than just a screenshot? Or maybe just demo for me when you're back in the office?

I'm guessing several of the messages (like `createdAppServicePlan`) flash up quickly without ever being readable. As a general rule, if the message doesn't end in `...`, I don't think it should be included in 'progress'.